### DISCUSSION TITLE
Can't access mastercontainer anymore (AIO Interface) & doesn't create other containers when redeployed

### DISCUSSION THREAD

**Discussion by Fred-DTV** [nextcloud/all-in-one#2694] [Questions] [answered]
Hi,

I am having serious issues with my AIO Instance on docker on ubuntu 22.04. Yesterday I realised that I couldn't access the AIO interface anymore and when I had a look in docker I found three issues: 
1. the apache container was not there anymore (however nextcloud was still reachable)
2. https://github.com/nextcloud/all-in-one/discussions/1970 this error in the mastercontainer, which I was able to remove thanks to this thread
3. INF | ts=1686121842.1104443 msg=using provided configuration config_file=/Caddyfile config_adapter=  This error which still persists in the mastercontainer

I have tried several ways to get it back up including:
1. Restarting the whole system
2. Removing all containers and repulling from the compose file. But thanks to this it is even worse now, because the mastercontainer doesn't spawn the other containers anymore and our whole NC is down now :/ (this is the compose file: https://pastebin.com/BaaYGUh4 and I tried with and without the dns setting)
3. I also tried this https://github.com/nextcloud/all-in-one/discussions/2065

Here is the ful log of the Mastercontainer:

`Trying to fix docker.sock permissions internally...
Adding internal www-data to group ping
Initial startup of Nextcloud All-in-One complete!
You should be able to open the Nextcloud AIO Interface now on port 8080 of this server!
E.g. https://internal.ip.of.this.server:8080

If your server has port 80 and 8443 open and you point a domain to your server, you can get a valid certificate automatically by opening the Nextcloud AIO Interface via:
https://your-domain-that-points-to-this-server.tld:8443
++ head -1 /mnt/docker-aio-config/data/daily_backup_time
+ BACKUP_TIME=04:00
+ export BACKUP_TIME
+ export DAILY_BACKUP=1
+ DAILY_BACKUP=1
++ sed -n 2p /mnt/docker-aio-config/data/daily_backup_time
+ '[' '' '!=' automaticUpdatesAreNotEnabled ']'
+ export AUTOMATIC_UPDATES=1
+ AUTOMATIC_UPDATES=1
+ set +x
INF | ts=1686122541.2007215 msg=using provided configuration config_file=/Caddyfile config_adapter= 
[07-Jun-2023 07:22:21] NOTICE: fpm is running, pid 143
[07-Jun-2023 07:22:21] NOTICE: ready to handle connections
[07-Jun-2023 07:24:30] NOTICE: Terminating ...
[07-Jun-2023 07:24:30] NOTICE: exiting, bye-bye!
Initial startup of Nextcloud All-in-One complete!
You should be able to open the Nextcloud AIO Interface now on port 8080 of this server!
E.g. https://internal.ip.of.this.server:8080

If your server has port 80 and 8443 open and you point a domain to your server, you can get a valid certificate automatically by opening the Nextcloud AIO Interface via:
https://your-domain-that-points-to-this-server.tld:8443
++ head -1 /mnt/docker-aio-config/data/daily_backup_time
+ BACKUP_TIME=04:00
+ export BACKUP_TIME
+ export DAILY_BACKUP=1
+ DAILY_BACKUP=1
++ sed -n 2p /mnt/docker-aio-config/data/daily_backup_time
+ '[' '' '!=' automaticUpdatesAreNotEnabled ']'
+ export AUTOMATIC_UPDATES=1
+ AUTOMATIC_UPDATES=1
+ set +x
INF | ts=1686122674.5010486 msg=using provided configuration config_file=/Caddyfile config_adapter= 
[07-Jun-2023 07:24:34] NOTICE: fpm is running, pid 130
[07-Jun-2023 07:24:34] NOTICE: ready to handle connections
`

I just don't know what to try anymore :/

**Comment by szaimen** [comment]
Hi, can you provide the mastercontainer logs from when you try to stop and start the containers from the AIO interface?

**Reply by Fred-DTV** [reply]
I can't access the AIO interface :/ I am starting and stopping through docker/portainer

**Reply by szaimen** [reply]
Ah I see. Cant you even use `https://ip.address.of.this.server:8080` ?

**Reply by Fred-DTV** [reply]
Ok I have found the issue why I couldn't connect anymore: I was out of office yesterday and was connecting via Wireguard VPN into our network.
However, the weird thing is that I was able to access other services on the server but AIO was the only one I couldn't access. 
Is there some kind of IP range allow list for the AIO interface?

At the moment the containers are updating and starting to 6.0.0.
My guess is that something went wrong during the update to 6.0.0  and that's when the Apache container went missing.

I will post another update when everything has booted up again

**Comment by szaimen** [comment, marked as answer]
> Ok I have found the issue why I couldn't connect anymore: I was out of office yesterday and was connecting via Wireguard VPN into our network.

Nice that you found the problem!

> However, the weird thing is that I was able to access other services on the server but AIO was the only one I couldn't access.
> Is there some kind of IP range allow list for the AIO interface?

No, there is not. I guess wireguard is somehow incorrectly configured then. That only the mastercontainer is making problems is just a coincidence. However I think there have been other threads about this. See https://github.com/nextcloud/all-in-one/discussions?discussions_q=is%3Aopen+wireguard+. Maybe it helps!

**Reply by Fred-DTV** [reply]
Ok everything is up and running again and the update to 6.0.0 went smoothly! 
Thanks for you work on this again. Wish I could be more of a help than just opening issues 🙈

Two last questions:
1. Would it make sense to raise the wmem buffer limit as well https://www.tecchannel.de/a/tcp-ip-tuning-fuer-linux,429773,6 ?
2. My collabora Container is running and seems to be working well according tro the Admin settings, but I can't find the office apps anywhere? 
![Screenshot 2023-06-07 at 11 24 27](https://github.com/nextcloud/all-in-one/assets/70434961/6a3cdb74-d7c2-4309-b383-4f864850a366)

**Reply by szaimen** [reply]
> * Would it make sense to raise the wmem buffer limit as well https://www.tecchannel.de/a/tcp-ip-tuning-fuer-linux,429773,6 ?

Yes, this is recommended for caddy: https://github.com/nextcloud/all-in-one/discussions/1970

> * My collabora Container is running and seems to be working well according tro the Admin settings, but I can't find the office apps anywhere?

It will not be listed in the apps list. It is a plugin for the files app when you open an office document. The main indication is that you have an admin setting for Office.

**Reply by Fred-DTV** [reply]
> > * Would it make sense to raise the wmem buffer limit as well https://www.tecchannel.de/a/tcp-ip-tuning-fuer-linux,429773,6 ?
> 
> Yes, this is recommended for caddy: #1970
> 
I meant the other buffer limit? because in that thread it only raises the rmem not the wmem?

> > * My collabora Container is running and seems to be working well according tro the Admin settings, but I can't find the office apps anywhere?
> 
> It will not be listed in the apps list. It is a plugin for the files app when you open an office document. The main indication is that you have an admin setting for Office.

Ok got it thanks a lot!

**Reply by szaimen** [reply]
> > > * Would it make sense to raise the wmem buffer limit as well https://www.tecchannel.de/a/tcp-ip-tuning-fuer-linux,429773,6 ?
> > 
> > 
> > Yes, this is recommended for caddy: #1970
> 
> I meant the other buffer limit? because in that thread it only raises the rmem not the wmem?

Honesty I never heard of the other setting. Only the first one is needed for caddy IIRC
